lib/application.ts

```diff
@@ -1445,6 +1452,14 @@ export class SNApplication {
     this.services.push(this.actionsManager!);
   }
 
+  private createNamespaceService() {
+    this.namespaceService = new SNNamespaceService(this.deviceInterface!);
+    if (this.fixedNamespace) {
+      this.namespaceService!.createFixedNamespace(this.fixedNamespace);
```

Comment:
I'm not yet sure how fixedNamespace functions, but sounds like this should be passed to the constructor instead, if it's something that shouldn't be called more than once.